BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsEventParams.kt

```diff
@@ -0,0 +1,18 @@
+package com.braintreepayments.api
+
```

Comment:
We should add the `RestrictTo` annotation here to keep this class internal to the BT SDK:

```kotlin
@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Reply:
Addressed: [229c16b](https://github.com/braintree/braintree_android/pull/1031/commits/229c16bdaae38efcec18775a69b1c753c5e8715a)